addons/website_event_questions/models/event_event.py

```diff
@@ -19,9 +19,9 @@ class EventEvent(models.Model):
 
```

Comment:
I had to read 3 times the commit message and read the diff to understand it. :D 
Here is a proposition : 
```
[IMP] website_event_questions: rename is_individual and inverse its behaviour

This commit inverse the behaviour of boolean 'is_individual' and rename it to 'once_per_order'.
Before this commit, default behaviour was to ask the event question for each attendee.
After this commit, default behaviour is to ask only once the question per order.
```